core/server/api/v2/utils/serializers/output/pages.js

```diff
@@ -1,21 +1,38 @@
+const _ = require('lodash');
 const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:pages');
 const url = require('./utils/url');
+const date = require('./utils/date');
+
+const mapPage = (model, frame) => {
+    const jsonModel = model.toJSON(frame.options);
+    url.forPost(model.id, jsonModel, frame.options);
```

Comment:
Either we raise an issue or we change this behaviour as separate commit.

modification by ref vs modification by copy (we mix both)